backend/src/users/user-relationship/user-relationship.controller.ts

```diff
@@ -47,6 +49,14 @@ export class UserRelationshipController {
 		});
```

Comment:
realtionshipData = relationshipData?
just a typo, but who knows it'll cause magical problems later

Reply:
Good spot!